pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -369,6 +369,17 @@ message LinuxContainerConfig {
     optional Capability capabilities = 2;
     // Optional SELinux context to be applied.
     optional SELinuxOption selinux_options = 3;
+    // User contains the user for the container process.
+    optional LinuxUser user = 4;
```

Comment:
@vishh the member is called `user`,  `LinuxUser` is the type defined outside `LinuxContainerConfig` so I'm not sure we want that as just `User` (as said we may have other platform dependant user struct). I'll change the definition if you really want to though.